controller/embedded/api-schema/entities/config-type.post.json

```diff
@@ -9,6 +9,13 @@
     "name": {
       "$ref": "definitions.all.json#/definitions/name"
     },
+    "schema": {
+      "type": [
+        "object",
+        "null"
+      ],
+      "default": null
```

Comment:
IIRC the JSON schema parser used in GO doesn't actually use these values. The Node.js one did.

They don't hurt and are part of the spec. Most of the time the GO JSON parser defaults to an empty value such as NIL but other times won't if the empty value is a non-nil value ("", 0, etc)

Reply:
I've noticed that with the API schemas, that I had to manage defaults manually in the code. 

If we start to care about it more, we can probably put together a PR to fix in the library.

Reply:
I vaguely recall the author saying "no" to a direct ask for that.

Reply:
I found it: https://github.com/xeipuuv/gojsonschema/issues/45 (with your comment :) )

Sounds like the maintainer didn't have time to pull that PR into shape, but would be ok with someone else doing it.

Reply:
So not a "no" just a "not me".